packages/terra-site/src/cerner-mock-theme.scss

```diff
@@ -23,6 +23,15 @@
   --terra-badge-background-color-info: #bdc3c7;
   --terra-badge-color-info: #333;
 
+  // terra-base
```

Comment:
We need to remove the quotes from these mock values in order to see the mock theme changes.

Reply:
Updated https://github.com/cerner/terra-core/pull/1171/commits/25ca61696f48d9b02c3b0df02332ba6f310ee170#diff-5b9ce4807ebb95c1f469f98b943515cfL27